Makefile

```diff
@@ -103,10 +103,10 @@ endif
 build: fleet fleetctl
 
 fleet: .prefix .pre-build .pre-fleet
-	go build -i -o build/${OUTPUT} -ldflags ${KIT_VERSION} ./cmd/fleet
+	GO111MODULE=on go build -mod vendor -i -o build/${OUTPUT} -ldflags ${KIT_VERSION} ./cmd/fleet
```

Comment:
`GO111MODULE=on` would be better set on the top level of the Makefile, akin to https://github.com/kolide/launcher/blob/master/Makefile#L11

Reply:
Why vendor here?

Reply:
Mainly to mimic the existing behavior with `dep`. I'll remove 😄